ShipEngineSDK/ShipEngineResponse.cs

```diff
@@ -0,0 +1,18 @@
+namespace ShipEngineSDK;
```

Comment:
Just a formatting beef, but I think most of our code puts the namespace right above the class declaration and after the `using` statements (even though I know some tools don't like doing this)